lstchain/data/lstchain_src_dep_config.json

```diff
@@ -148,7 +150,7 @@
     "dist"
   ],
 
-  "particle_classification_features": [
+  "disp_classification_features": [
```

Comment:
What does "from_source" mean in this context? We don't know the source position yet at this point.
Ah wait, this is source-dependent... then why disp reconstruction??

Reply:
Now I'm wondering if the information of reconstructed disp can be used for source-dependent analysis (especially at high energy ranges).
So I want to keep the disp reconstruction procedure even for source-dependent analysis.
(But input parameters would be irrelevant indeed for the moment...)

Reply:
ok!